diff --git a/Makefile.options b/Makefile.options
index 08e858b..110f784 100644
--- a/Makefile.options
+++ b/Makefile.options
@@ -14,10 +14,10 @@ LUAC_BIN = luac
 # C compiler flags
 # TODO: check alternatives to -D_GNU_SOURCE -fPIC
 #       using libtool?
-CFLAGS = -O2 -D_GNU_SOURCE -fPIC -Wall -I /usr/include -I /usr/local/include -I /usr/local/include/lua52
+CFLAGS = -O2 -D_GNU_SOURCE -fPIC -Wall -I /usr/include -I /usr/include/lua5.1
 
 # additional C compiler flags for parts which depend on PostgreSQL
-CFLAGS_PGSQL = -I /usr/local/include/postgresql -I /usr/local/include/postgresql/server
+CFLAGS_PGSQL = -I /usr/include/postgresql -I /usr/include/postgresql/9.4/server
 
 # linker flags
 LDFLAGS = -shared -L /usr/lib -L /usr/local/lib
